frontend/src/routes/home/home.tsx

```diff
@@ -77,56 +34,85 @@ export const Home = () => {
   }
 
   return (
-    <Splash data-testid='home-splash' style={{ textAlign: 'center' }}>
-      <Header
+    <div
+      data-testid='wallet-home'
+      className='vega-border-image'
+      style={{ borderTop: '3px solid', padding: 20 }}
+    >
+      <Title
         style={{
           margin: '0 0 30px 0',
-          color: Colors.WHITE,
-          textAlign: 'center'
+          color: Colors.WHITE
         }}
       >
-        <Vega />
-      </Header>
-      {wallets.length > 0 && (
-        <>
-          <ButtonGroup orientation='vertical'>
-            {wallets.map(w => (
-              <Button
-                onClick={() => {
-                  dispatch(actions.getKeysAction(w.name))
-                }}
-                data-testid={`wallet-${w.name.replace(' ', '-')}`}
-                key={w.name}
-              >
-                {w.name}
-              </Button>
-            ))}
-          </ButtonGroup>
-          <p style={{ margin: '20px 0', textAlign: 'center' }}>OR</p>
-        </>
-      )}
-      <ButtonGroup orientation='vertical' style={{ marginBottom: 20 }}>
-        <Link to='/wallet-create'>
-          <Button data-testid='create-new-wallet' style={{ width: '100%' }}>
-            Create wallet
-          </Button>
-        </Link>
-        <Link to='/wallet-import'>
-          <Button data-testid='import-wallet' style={{ width: '100%' }}>
-            Import wallet
-          </Button>
-        </Link>
-      </ButtonGroup>
-      <p>
-        <button
-          style={{ textDecoration: 'underline' }}
-          onClick={() => dispatch({ type: 'SET_SETTINGS_MODAL', open: true })}
-          data-testid='home-settings'
+        Wallets
+      </Title>
+      <div
+        style={{
+          paddingBottom: 144,
+          width: '100%'
+        }}
+      >
+        <div
+          style={{
+            borderBottom: `${wallets.length > 0 ? '1' : '0'}px solid ${
+              Colors.BLACK
+            }`
+          }}
         >
-          App settings
-        </button>
-      </p>
+          {wallets.map(w => (
+            <div
+              style={itemStyles}
+              onClick={() => open(w.name)}
+              data-testid={`wallet-${w.name.replace(' ', '-')}`}
+              key={w.name}
+            >
+              <div>{w.name}</div>
+              <div style={{ color: Colors.GRAY_1 }}>
+                <Lock style={{ width: 20, margin: '0 20px' }} />
+              </div>
+            </div>
+          ))}
+        </div>
+      </div>
+      <div
+        style={{
+          textAlign: 'center',
+          padding: 20,
+          width: '100%',
+          backgroundColor: Colors.DARK_GRAY_1,
+          ...(wallets.length
```

Comment:
Suggestion(non-blocking): declare separate and the spread

Reply:
changed.